### PR TITLE
feat/plugins-rate-limiting by path. 

### PR DESCRIPTION
**Pull request by fairyqb**
**Summary**

add plugins-rate-limiting limit by path.

### CODE REVIEW

**Comment by bungle** [comment]
Can you please rebase this on `next branch` as new `features` goes to `next` branch, thank you.

**Comment by fairyqb** [comment]
OK, #6263

**Comment by fairyqb** [comment]
> Can you please rebase this on `next branch` as new `features` goes to `next` branch, thank you.

@bungle  @Tieske 
sorry, rebase completed now

File: kong/plugins/rate-limiting/schema.lua [hunk 1]
```diff
@@ -39,9 +39,10 @@ return {
           { limit_by = {
               type = "string",
               default = "consumer",
-              one_of = { "consumer", "credential", "ip", "service", "header" },
+              one_of = { "consumer", "credential", "ip", "service", "header", "path" },
           }, },
           { header_name = typedefs.header_name },
+          { path = typedefs.path },
```
Comment (dndx) [inline kong/plugins/rate-limiting/schema.lua:45]: The above code does not make use of this config at all and should probably be removed.

File: kong/plugins/rate-limiting/handler.lua [hunk 2]
```diff
@@ -67,6 +67,9 @@ local function get_identifier(conf)
 
   elseif conf.limit_by == "header" then
     identifier = kong.request.get_header(conf.header_name)
+
+  elseif conf.limit_by == "path" then
+    identifier = kong.request.get_path(conf.path)
```
Comment (dndx) [inline kong/plugins/rate-limiting/handler.lua:72]: `kong.request.get_path` does not expect any arguments, so `conf.path` is not used at all here.

**Review by dndx — CHANGES_REQUESTED** [review]

**Comment by kikito** [comment]
Merged with fixes in #6286. Thanks!

**Pull request closed without merging**